### PR TITLE
itvr-311- add model 

### PR DESCRIPTION
**Pull request by vibhiquartech**
https://apps.nrs.gov.bc.ca/int/jira/browse/CTHUB-311

### CODE REVIEW

File: django/api/models/rebate_historical.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+)
+
+from django.core.validators import MinLengthValidator
+
+from django_extensions.db.models import TimeStampedModel
+from django.utils.translation import gettext_lazy as _
+from ..models import GoElectricRebateApplication
+
+
+class rebate_historical(TimeStampedModel):
```
Comment (naomiaro) [inline django/api/models/rebate_historical.py:19]: We should follow classname convention. This won't just be historical rebates so maybe we can call this class `GoElectricRebate`

`django/api/migrations/0001_initial.py`


We've been deleting this for now and recreating the migrations using

`python manage.py makemigrations api`

in the docker container.


We've then been making a model export in this file:

`django/api/models/__init__.py`

File: django/api/models/rebate_historical.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    id = ShortUUIDField(length=16, primary_key=True, editable=False)
+    application_id = ForeignKey(GoElectricRebateApplication, on_delete=PROTECT)
```
Comment (naomiaro) [inline django/api/models/rebate_historical.py:21]: Does this allow for null values?

File: django/api/models/rebate_historical.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+from django.core.validators import MinLengthValidator
+
+from django_extensions.db.models import TimeStampedModel
+from django.utils.translation import gettext_lazy as _
+from ..models import GoElectricRebateApplication
+
+
+class rebate_historical(TimeStampedModel):
+    id = ShortUUIDField(length=16, primary_key=True, editable=False)
```
Comment (naomiaro) [inline django/api/models/rebate_historical.py:20]: We don't need a short uuid here for the rebates, we could prob use a regular auto incrementing id here.

**Review by naomiaro — CHANGES_REQUESTED** [review]

File: django/api/models/rebate_historical.py (hunk 2 above)
Comment (vibhiquartech) [reply django/api/models/rebate_historical.py:21]: @naomiaro Unless specified with null=True,  null values will not be allowed by default. Should we allow null values for application_id field?

Comment (naomiaro) [reply django/api/models/rebate_historical.py:21]: Yes the historical applications won't have an application in our system, just new records we put in this table.

**Review by naomiaro — APPROVED** [review]

**Pull request merged**